internal/pkg/cli/pipeline_show.go

```diff
@@ -163,7 +163,10 @@ func (o *showPipelineOpts) askPipelineName() error {
 	}
 
 	if len(pipelineNames) == 1 {
-		o.pipelineName = pipelineNames[0]
+		pipelineName = pipelineNames[0]
+		log.Infof("Found pipeline: %s\n.", color.HighlightUserInput(pipelineName))
```

Comment:
Note: Don't have to keep this log statement, but it helps see that the thing works in the manual output for now.